src/Orleans/Properties/AssemblyInfo.cs

```diff
@@ -26,6 +26,7 @@
 [assembly: InternalsVisibleTo("OrleansProviders")]
 [assembly: InternalsVisibleTo("OrleansHost")]
 [assembly: InternalsVisibleTo("OrleansAzureUtils")]
+[assembly: InternalsVisibleTo("OrleansServiceBus")]
```

Comment:
3.use a third party timer. Is there a timer class in .net which can be configured to fire events periodically ? it's totally possible that it is there but I just don't know.